tests/basilisp/importer_test.py

```diff
@@ -99,10 +99,13 @@ def module_cache(self):
     @pytest.fixture
     def module_dir(self, monkeypatch: MonkeyPatch, module_cache):
         with TemporaryDirectory() as tmpdir:
+            cwd = os.getcwd()
             monkeypatch.chdir(tmpdir)
             monkeypatch.syspath_prepend(tmpdir)
             monkeypatch.setattr("sys.modules", module_cache)
             yield tmpdir
+            monkeypatch.chdir(tmpdir)
```

Comment:
What is the purpose of this `chdir`?

Reply:
Indeed, this was a mistake and thus removed.